terrautils/extractors.py

```diff
@@ -9,6 +9,7 @@
 import os
 import requests
 from urllib3.filepost import encode_multipart_formdata
+import logstash
```

Comment:
Not sure we need to actually import logstash, this is done using the logstash.TCPLogstashHandler